stream-compositions/api/service-api/product-composition-api/src/main/resources/service-api-v2.yaml

```diff
@@ -99,6 +99,8 @@ components:
           type: string
         userInternalId:
           type: string
+        source:
```

Comment:
we should include a description in this property so whoever uses this spec can understand what this is about, preferable with examples too